tests/views/graph_manager_tests.py

```diff
@@ -470,3 +471,18 @@ def test_graph_import(self):
         self.assertEqual(imported_json[0], [])
         self.assertEqual(imported_json[1]["graphs_saved"], 1)
         self.assertEqual(imported_json[1]["name"], "Cardinality Test Model")
+
+    def test_prevent_card_save_of_published_graph(self):
+        """
+        test to make sure that a card cannot be saved if the graph is published
```

Comment:
I usually chop these if the test title is good enough (it's A+).

While here, it seems weird that the coverage check failed even though you added a test. I'll run the coverage check locally and see if I get the same result.

Reply:
Ah, it's because the view is not tested, and this is only testing the model. Since this test file is under `tests/views`, do you think it makes more sense to test the view? I find it handy to `self.assertContains, response, "error msg", status_code=403`.